book/src/programmability/events.md

```diff
@@ -13,7 +13,7 @@ on-chain. Events are emitted by the `sui::event` module located in the
 // File: sui-framework/sources/event.move
 module sui::event;
 
-/// Emit a custom Move event, sending the data offchain.
+/// Emit a custom Move event, sending the data off-chain.
 ///
 /// Used for creating custom indexes and tracking onchain
```

Comment:
```suggestion
/// Used for creating custom indexes and tracking on-chain
```
not sure if this is consistent as is across the doc set but should matchy matchy with L16 at least.